src/client.js

```diff
@@ -118,6 +119,15 @@ class Client {
   }
 
   /**
+   * A wrapper around the constructor, for backwards compatability.
```

Comment:
Do you mean passing a `Config` object to `Client` class?

Most vanilla javascript can't handle the `import` keyword, so we need a globally accessible entry point. For the purposes of compilation, it's nice to have a single entry point as a file so we can take advantage of tree shaking. And we have one external interface, so that if we need to change things internally, we can just change the implementation of `buildClient` to fit what lives under the hood.

Does that answer what you're asking? I'm not sure I understand.